services/api/src/index.ts

```diff
@@ -487,21 +604,31 @@ server.after(() => {
             response: { 200: Dto.checkLoginStatusResponse },
         },
         handler: async (request) => {
-            const { didWrite } = await verifyUCAN(db, request, {
-                expectedDeviceStatus: undefined,
-            });
-
-            const status = await authUtilService.getDeviceStatus(db, didWrite);
-            const loggedInStatus: DeviceLoginStatus = !status.isRegistered
-                ? "unknown"
-                : !status.isVerified
-                ? "unverified"
-                : !status.isLoggedIn
-                ? "logged_out"
-                : "logged_in";
-            return {
-                loggedInStatus: loggedInStatus,
-            };
+            const { deviceStatus } = await verifyUcanAndDeviceStatus(
```

Comment:
This function doesn't assume the device is "known", as a result the "userId" in deviceStatus may be undefined. The difference with "verifyUcan" is that it returns the deviceStatus.